subworkflows/local/variant_calling/call_sv_manta.nf

```diff
@@ -0,0 +1,54 @@
+//
+// A structural variant caller workflow for manta
+//
+
+include { MANTA_GERMLINE as MANTA } from '../../../modules/nf-core/manta/germline/main'
+
+workflow CALL_SV_MANTA {
+    take:
+        ch_bam       // channel: [mandatory] [ val(meta), path(bam) ]
+        ch_bai       // channel: [mandatory] [ val(meta), path(bai) ]
+        ch_fasta     // channel: [mandatory] [ path(fasta) ]
+        ch_fai       // channel: [mandatory] [ path(fai) ]
+        ch_case_info // channel: [mandatory] [ val(case_info) ]
+        ch_bed       // channel: [mandatory for WES] [ val(meta), path(bed), path(tbi) ]
+
+    main:
+        ch_bam.collect{it[1]}
+            .toList()
+            .set { bam_file_list }
+
+        ch_bai.collect{it[1]}
+            .toList()
+            .set { bai_file_list }
+
+        ch_bed.map {
+                id, bed_file, index ->
+                    return [bed_file, index]}
+            .set { bed_input }
+
+        if (params.analysis_type.toUpperCase() == "WGS" ) {
```

Comment:
you should not need to use `toUpperCase` as this params can be checked with the json schema

Reply:
Addressed here https://github.com/nf-core/raredisease/pull/348/files